common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -724,6 +727,8 @@ function (HTML5Video, Resizer) {
             case this.videoPlayer.PlayerState.BUFFERING:
                 this.el.addClass('is-buffered');
                 this.el.trigger('buffering');
+                this.videoPlayer
+                    .emitPlayVideoEvent = this.videoPlayer.lastPlayerState !== this.videoPlayer.PlayerState.PLAYING;
```

Comment:
As I noted above this is not the right solution. For the following sequence of events: buffering, playing, buffering, playing, paused, playing, the last state change should trigger a play_video event.